man/docker-run.1.md

```diff
@@ -421,6 +422,10 @@ Use `docker port` to see the actual mapping: `docker port CONTAINER $CONTAINERPO
      **host**: use the host's PID namespace inside the container.
      Note: the host mode gives the container full access to local PID and is therefore considered insecure.
 
+**--userns**=*host*
```

Comment:
Thanks, fixed. 
Note that the `--pid` option is also specified with `host` as default (this probably needs to be changed).